### PR TITLE
[DO NOT MERGE] Technical PR to review rebranding

### PR DESCRIPTION
**Pull request by alongosz**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -30,34 +28,61 @@
...
+        "phpstan/phpstan-symfony": "^1.0"
     },
     "autoload": {
         "psr-4": {
             "EzSystems\\EzPlatformAutomatedTranslation\\": "lib",
             "EzSystems\\EzPlatformAutomatedTranslationBundle\\": "bundle",
-            "EzSystems\\EzPlatformAutomatedTranslationBundle\\Tests\\": "tests"
+            "Ibexa\\AutomatedTranslation\\": "src/lib/",
+            "Ibexa\\Bundle\\AutomatedTranslation\\": "src/bundle/",
+            "Ibexa\\Contracts\\AutomatedTranslation\\": "src/contracts/"
```
Comment (alongosz) [inline composer.json:50]: Those should go on top, given it's the physical namespace used in the files, while EzSystems is just an alias

File: src/bundle/Form/TranslationAddDataTransformer.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+use Symfony\Component\Form\DataTransformerInterface;
+
+class TranslationAddDataTransformer implements DataTransformerInterface
+{
+    /**
+     * @param \Ibexa\AdminUi\Form\Data\Content\Translation\TranslationAddData $value
+     *
+     * @return \Ibexa\Bundle\AutomatedTranslation\Form\Data\TranslationAddData
+     */
+    public function transform($value)
```
Comment (adamwojs) [inline src/bundle/Form/TranslationAddDataTransformer.php:22]:
```suggestion
    public function transform($value): TranslationAddData
```

but implementation make a lot of optimistic assumptions about \`$value\`

File: src/bundle/Resources/config/adminui.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+services:
+
+    ibexa.automated_translate.adminui.component.javascripts:
```
Comment (adamwojs) [inline src/bundle/Resources/config/adminui.yml:3]:
```suggestion
services:
    ibexa.automated_translate.adminui.component.javascripts:
```

Comment (alongosz) [inline src/bundle/Resources/config/adminui.yml:3]: Hmm, naming - `automated_translate` has some grammar issue, but more importantly, where's the class for that service? I see only parent definition.

File: src/bundle/Resources/config/services.yml [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+            $blockDefinitionFactory: '@Ibexa\FieldTypePage\FieldType\Page\Block\Definition\BlockDefinitionFactory'
+
+    Ibexa\AutomatedTranslation\Encoder\Field\RichTextFieldEncoder:
+        arguments:
+            $richTextEncoder: '@Ibexa\AutomatedTranslation\Encoder\RichText\RichTextEncoder'
+
+    # field encoder manager
+    Ibexa\AutomatedTranslation\Encoder\Field\FieldEncoderManager:
+        arguments:
+            $fieldEncoders: !tagged_iterator 'ibexa.automated_translation.field_encoder'
```
Comment (alongosz) [inline src/bundle/Resources/config/services.yml:29]: quotes can be skipped, especially that there are places in this file where they're not used, so it's a bit inconsistent.

File: src/bundle/Resources/config/services_override.yml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+services:
+    Ibexa\Bundle\AutomatedTranslation\Controller\TranslationController:
+        decorates: Ibexa\Bundle\AdminUi\Controller\TranslationController
+        autoconfigure: true
+        autowire: true
+        tags: ['controller.service_arguments']
```
Comment (alongosz) [inline src/bundle/Resources/config/services_override.yml:6]: missing new line at the end of line

File: src/bundle/Resources/doc/FEATURES.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
+# Features
```
Comment (alongosz) [inline src/bundle/Resources/doc/FEATURES.md:1]: TBH the whole doc section should be dropped and moved to the official documentation. This has been a pattern for quite some time.

File: src/bundle/DependencyInjection/Configuration.php [hunk 7]
```diff
@@ -1,21 +1,21 @@
...
 use Symfony\Component\Config\Definition\Builder\TreeBuilder;
 
 class Configuration extends SiteAccessAware\Configuration
 {
     public function getConfigTreeBuilder(): TreeBuilder
     {
-        $treeBuilder = new TreeBuilder('ez_platform_automated_translation');
+        $treeBuilder = new TreeBuilder('ibexa_automated_translation');
         $rootNode = $treeBuilder->getRootNode();
         $systemNode = $this->generateScopeBaseNode($rootNode);
```
Comment (alongosz) [inline src/bundle/DependencyInjection/Configuration.php:20]: TBH I would prefer Parsers for `ibexa.system.<scope>.<settings>` to this old extension point, but probably out of scope for rebranding.

File: src/bundle/Resources/config/adminui.yml (hunk 3 above)
Comment (damianz5) [reply src/bundle/Resources/config/adminui.yml:3]: similar as https://github.com/ibexa/admin-ui/blob/main/src/bundle/Resources/config/services/dashboard.yaml#L47

**Comment by alongosz** [comment]
All addressed in #3, thanks!

**Pull request closed without merging**